Libraries/Boards/MAX32665/EvKit_129B/board.mk

```diff
@@ -33,6 +33,7 @@ SRCS += stdio.c
 SRCS += led.c
 SRCS += mx25.c
 SRCS += pb.c
+SRCS += adt7320_driver.c
```

Comment:
Usually, we would only want to add the source file to the BSP's if the temp sensor is included on the EVKIT itself.  Since it's external, I would suggest adding the following to `project.mk` instead.

```Makefile
VPATH += $(LIBS_DIR)/MiscDrivers/TempSensor
IPATH += $(LIBS_DIR)/MiscDrivers/TempSensor
SRCS += adt7320_driver.c
``` 

The reasoning behind this is that we don't want to compile extra files if they aren't going to be used.